zul/src/archive/web/js/zul/db/Calendar.js

```diff
@@ -833,7 +833,10 @@ zul.db.Calendar = zk.$extends(zul.Widget, {
 					width = oldMid.offsetWidth,
 					x = width * -1,
 					self = this,
-					animaCSS = this.$s('anima');
+					animaCSS = this.$s('anima'),
+					todayBtn = this.isShowTodayLink() ? jq(this.$n('today')).parent() : jq();
+
+				todayBtn.is(':hidden') && todayBtn.css('display', 'none');

```

Comment:
When animating, the hidden button will display for a while.
To solve this problem, I add the style="display: none" before the animation and strip it after the animation.

If the button is not hidden, just skip it.